clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/HealthState.groovy

```diff
@@ -2,5 +2,5 @@
 package com.netflix.spinnaker.clouddriver.model
 
 enum HealthState {
-  Up, Down, Unknown, Starting, OutOfService
+  Up, Down, Unknown, Starting, OutOfService, Succeeded, Failed
```

Comment:
Then again, maybe they shouldn't live in separate enums since the states are all still mutually exclusive.